src/domains/wallet-app/env/weu-uat/terraform.tfvars

```diff
@@ -35,8 +35,8 @@ tls_cert_check_helm = {
   image_tag     = "v1.3.4@sha256:c3d45736706c981493b6216451fc65e99a69d5d64409ccb1c4ca93fef57c921d"
 }
 
-payment_wallet_with_pm_enabled = true
+payment_wallet_with_pm_enabled = false
```

Comment:
is it right to disable wallet with PM in uat env?

Reply:
Yes, we are going to switch test on new apis